packages/tpetra/core/ext/TpetraExt_MatrixMatrix_def.hpp

```diff
@@ -3759,26 +3627,18 @@ addUnsorted(
   typename AddKern::KKH handle;
   handle.create_spadd_handle(false);
   auto addHandle = handle.get_spadd_handle();
-#ifdef HAVE_TPETRA_MMM_TIMINGS
-  auto MM = rcp(new TimeMonitor(*TimeMonitor::getNewTimer("TpetraExt::MatrixMatrix::add() unsorted symbolic")));
-#endif
+  auto MM = rcp(new Tpetra::Details::ProfilingRegion("TpetraExt: add: unsorted symbolic"));
+
   KokkosSparse::Experimental::spadd_symbolic
```

Comment:
I realize you didn't touch this, but should `spadd_symbolic` also be moved out of the Experimental namespace for the same reason as `spgemm_symbolic`?